pkgs/development/python-modules/cwl-upgrader/default.nix

```
@@ -11,7 +11,7 @@
 
 buildPythonPackage rec {
   pname = "cwl-upgrader";
-  version = "1.2.8";
+  version = "1.2.9";
   format = "setuptools";
 
   disabled = pythonOlder "3.7";
```

```suggestion
  disabled = pythonOlder "3.8";
```
https://github.com/common-workflow-language/cwl-upgrader/blob/6c50d8c5829dbdbad51e3b7691a0565fab7b2724/setup.py#L34

---

Thanks, updated.